docs/DeveloperGuide.md

```diff
@@ -308,7 +308,99 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case 6: View a student's details**
+
+Precondition: The student's details have been added to TutorAid.
+
+**MSS**
+
+1. Tutor requests a list of students
+
+2. TutorAid shows the list of students
+
+3. Tutor finds the specific student in the list and requests to view the student's details
+
+4. TutorAid shows the student's details
+
+   Use case ends.
+
+**Extensions**
+
+* 3a. The given index is invalid.
+
+  * 3a1. TutorAid shows an error message.
+
+    Use case resumes at step 3.
+
+**Use case 13: Set up a class**
```

Comment:
Maybe we can rename the use case to 'Set up a recurring class' since we plan to differentiate between a recurring and an ad-hoc class and the set up for an ad-hoc class may be different?

Reply:
Fixed, thanks